R/zzz.R

```diff
@@ -44,7 +44,12 @@ with_defaults <- function(..., default = default_linters) {
     # var[["foo"]]    => "foo"
     nms[missing] <- re_substitutes(
       re_substitutes(
-        re_substitutes(args, rex("(", anything), ""),
+        re_substitutes(
+          # Very long input might have newlines which are not caught
+          #  by . in a perl regex; see #774
+          gsub("\n", "", args, fixed = TRUE),
+          rex("(", anything), ""
+        ),
```

Comment:
could we stick to one style of regex-ing?
`re_substitutes(..., options = "s")` will enable dotall / single-line mode so that the regex doesn't need fixing.

Reply:
great! TIL